moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -268,9 +268,15 @@ const ProjectActivityLog = () => {
                         className={classes.tableCell}
                         style={{ whiteSpace: "nowrap" }}
                       >
-                        <span>{formatRelativeDate(change.created_at)}</span>
+                        <span>
+                          {change.created_at
+                            ? formatRelativeDate(change.created_at)
+                            : ""}
+                        </span>
                         <span className={classes.mutedDate}>
-                          {new Date(change.created_at).toLocaleString()}
+                          {change.created_at
+                            ? new Date(change.created_at).toLocaleString()
+                            : ""}
```

Comment:
there are a few projects in the access database that do not have a created date :/